webrender/res/brush_image.glsl

```diff
@@ -46,7 +52,7 @@ void brush_vs(
 vec4 brush_fs() {
     vec2 uv = clamp(vUv.xy, vUvBounds.xy, vUvBounds.zw);
 
-    vec4 color = texture(sColor0, vec3(uv, vUv.z));
+    vec4 color = TEX_SAMPLE(sColor0, vec3(uv, vUv.z));
```

Comment:
Actually, this would be incorrect, given that we want to use the mipmaps, and this macro always picks lod of 0.0

Reply:
Good catch, fixed.